src/Agent/NewRelic/Agent/Core/DataTransport/DataStreamingService.cs

```diff
@@ -762,11 +765,18 @@ private bool StreamRequests(PartitionedBlockingCollection<TRequest> collection,
 
                 _responseStreamsDic[consumerId] = new ResponseStreamWrapper<TResponse>(consumerId, responseStream, serviceAndStreamCancellationTokenSource.Token);
 
+                var hasUpdatedStreamingCount = false;
```

Comment:
This variable name kind of confused me at first. I think that you are trying to represent the concept that this consumer is now active or is currently streaming, which is similar to the previous `IsStreaming` property. Maybe we should rename this variable to something like `isStreaming`?

Reply:
renamed to isStreaming